brainfuck-dap/src/main.rs

```diff
@@ -289,6 +343,13 @@ struct DisconnectRequestArguments {
 /* ----------------- main ----------------- */
 
 fn main() {
+    CombinedLogger::init(vec![WriteLogger::new(
+        LevelFilter::Debug,
```

Comment:
Log level should be set according to environment variable.

Reply:
Fixed in latest version. Thanks for suggestion.